openstack/clustering/v1/webhooks/requests.go

```diff
@@ -0,0 +1,43 @@
+package webhooks
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+// TriggerOpts Webhooks request parameters
+type TriggerOpts struct {
+	// V (Optional)	query	string	The webhook implementation version requested.
+	V string `q:"V, omitempty"`
+
+	// params (Optional)	query	object	The query string that forms the inputs to use for the targeted action.
+	Params string `q:"params, omitempty"`
+}
+
+// TriggerOptsBuilder Query string builder interface for webhooks
+type TriggerOptsBuilder interface {
+	ToTriggerQuery() (string, error)
```

Comment:
`Query` is only for actions that utilize a URL query string. Since we're doing a POST, the Gophercloud convention is to use `Map` in the name. I think a better name for this function is:

```go
ToWebhookTriggerMap() (map[string]interface, err)
```

Reply:
Need to look into this one. Doc specified query string.

Reply:
Yes! Absolutely code/implementation over docs :smile: 

API docs are a good second reference but we can't use them as the primary reference since there have been numerous times when the docs did not match up with what was really happening in the code. (I've mentioned elsewhere that this is not criticism against people who write docs - having docs go out of sync with the code happens all the time)

I've also mentioned elsewhere that this makes implementing features in Gophercloud difficult -- there's no denying that one. I'm happy to help with this area whenever I can.

We use the service-side/api code as the primary reference but if that proves overly difficult, then the Python client code can also be used (in this case, `python-senlinclient`). As a last resort, running commands on the command-line with `--debug` set can also be helpful.

Reply:
Per discovery of where triggers are actually processed, this is definitely a Query, but the name of the function still needs 4 parts to conform with the rest of Gophercloud. Let's go with `ToWebhookTriggerQuery`